src/actions/index.js

```diff
@@ -0,0 +1,19 @@
+const actions = {
+    RESET_BANEER_STATE: 'RESET_BANEER_STATE',
+    RESET_CART_DATA: 'RESET_CART_DATA',
+    CART_PROCESS_RESET:'CART_PROCESS_RESET',
+    GET_BANNER_DETAILS: 'GET_LAUNCH_DATA',
+    GET_BANNER_DETAILS_SUCCESS: 'GET_LAUNCH_DATA_SUCCESS',
+    GET_BANNER_DETAILS_FAILURE: 'GET_LAUNCH_DATA_FAILURE',
+    GET_CATEGORY_DETAILS: 'GET_CATEGORY_DETAILS',
+    GET_CATEGORY_DETAILS_SUCCESS: 'GET_CATEGORY_DETAILS_SUCCESS',
+    GET_CATEGORY_DETAILS_FAILURE: 'GET_CATEGORY_DETAILS_FAILURE',
+    GET_PRODUCT_DETAILS: 'GET_PRODUCT_DETAILS',
+    GET_PRODUCT_DETAILS_SUCCESS: 'GET_PRODUCT_DETAILS_SUCCESS',
+    GET_PRODUCT_DETAILS_FAILURE: 'GET_PRODUCT_DETAILS_FAILURE',
+    POST_CART_DATA: 'POST_CART_DATA',
+    POST_CART_DATA_SUCCESS: 'POST_CART_DATA_SUCCESS',
+    POST_CART_DATA_FAILURE: 'POST_CART_DATA_FAILURE',
+    SET_USER: 'SET_USER'
+}
+export default actions;
```

Comment:
add a line break

Reply:
given line break before export default

Reply:
you have to add line break at the end of the file

Reply:
done